compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/Utils.desktop.kt

```diff
@@ -88,3 +89,16 @@ internal fun JComponent.setTransparent(transparent: Boolean) {
         isOpaque = false
     }
 }
+
+/**
+ * Calls [block] synchronously on the event dispatching thread. If the calling thread is already the
+ * event dispatch thread, simply executes [block]; otherwise schedules [block] to run on event
+ * dispatching thread and waits for it to return.
+ */
+fun runOnUiThread(block: () -> Unit) {
```

Comment:
```suggestion
internal fun runOnEDTThread(block: () -> Unit) {
```
or `runOnAwtThread`, just to emphasise that it should be used only with AWT implementation.